arborist/user.go

```diff
@@ -150,6 +150,56 @@ func userWithName(db *sqlx.DB, name string) (*UserFromQuery, error) {
 	return &user, nil
 }
 
+type PolicyInfoFromQuery struct {
+	Name           string `db:"name"`
+	Authz_provider string `db:"authz_provider"`
+	Type           string `db:"type"`
```

Comment:
~could you document the values `type` can take and what they mean~

nvm, see my other comment